README.md

```diff
@@ -70,6 +70,35 @@ avoids issues with encoding, as well as issues of information leaking. The
 designation of SHA1 is arbitrary and seen as a general "good enough" assertion
 of the value.
 
+### Tar form
+
+```mtree
+# .
+/set type=file mode=0664 uid=1000 gid=1000
+. type=dir mode=0775 tar_time=1468430408.000000000
+
+# samedir
+samedir type=dir mode=0775 tar_time=1468000972.000000000
+    file2 size=0 tar_time=1467999782.000000000
+    file1 size=0 tar_time=1467999781.000000000
+    
+[...]
+```
+
+While `go-mtree` serves mainly as a library for upstream `mtree` support,
+`go-mtree` is also compatible with [tar archives][tar] (which is not an upstream feature).
+This means that we can now create and validate a manifest by specifying a tar file.
```

Comment:
perhaps mention that one could produce a manifest from an archive, and then validate it against an on disk directory